ShareExtension/ShareExtension/Views/HomeView.swift

```diff
@@ -8,10 +8,54 @@
 import SwiftUI
 
 struct HomeView: View {
+    private var columns: [GridItem] = []
+    private let rowSpacing: CGFloat
+
+    public init(
+        columnSpacing: CGFloat = 10,
+        numberOfColumns: Int = 3,
+        rowSpacing: CGFloat = 10
+    ) {
+        columns.append(
+            contentsOf: repeatElement(
+                GridItem(
+                    .flexible(),
+                    spacing: columnSpacing
+                ),
+                count: numberOfColumns
+            )
+        )
+        self.rowSpacing = rowSpacing
+    }
+
+    @State private var isImagePresented: Bool = false
```

Comment:
Declare all of the properties at the beginning of the file for better structure and readable code and make priority to first be the "@Wrapper" properties and the the normal ones below the wrapped properties.

I.e.
 struct Example {
@State var state: String
@State var bind: String

var variable: Int
var type: String

.... rest of the code
}